turtle-package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
what is this used for?

Reply:
is packed inside shellapp as universe-package.json and later is replacing packge.json in turtle build
https://github.com/expo/turtle/blob/master/Dockerfile#L104

Reply:
Removable when we merge https://github.com/expo/expo/pull/4082.